pages/landing/patcher/use-cases/index.html

```diff
@@ -0,0 +1,68 @@
+---
+layout: landing
+type: landing
+slug: index-page
+
+extra_nav_links:
+  - name: Use Cases
+    url: /patcher/use-cases/
+
+title: Patcher Use Cases
+excerpt: Some examples of scenarios where Patcher can be useful
+permalink: /patcher/use-cases/
+
+use_cases:
+  - heading: "Use case 1: Update to Terraform 1.0 automatically"
+    tab_block: site.data.patcher-landing-page-tabs-terraform-upgrade
+    body: |
+      <ul>
+        <li>Use Patcher to update your Terraform code from an older version (e.g. 0.11) all the way up to the latest version (e.g. 1.0) automatically.</li>
+        <li>Patcher does this using <i>Patch Files</i>, which transform code automatically (e.g. switching from HCL1 to HCL2 syntax or updating <code>source</code> URLs to compatible module versions) and open pull requests with the results.</li>
+        <li>You can use off-the-shelf Patch Files created by Gruntwork (e.g. we offer Terraform upgrade patches such as <code>tf-13-to-14.patch</code>, <code>tf-14-to-15.patch</code>, etc) or create your own Patch Files!</li>
+      </ul>
+
+  - heading: "Use case 2: Update to React 17.0 automatically"
```

Comment:
This one is very similar to the Terraform one. I like that it shows the same approach works with many technologies, but listing it as the second one doesn't have as much "umph." Let's make this the last one.

Reply:
Done in https://github.com/gruntwork-io/gruntwork-io.github.io/pull/583/commits/115009de8ad7a06a94acf4af5eab4d942cbf5ac5.